.storybook/webpack.config.js

```diff
@@ -24,8 +24,8 @@ const custom = require('../config/webpack/webpack.common')({
 module.exports = ({config}) => {
     config.resolve.alias = {
         'react-native-config': 'react-web-config',
-        'react-native$': '@expensify/react-native-web',
-        'react-native-web': '@expensify/react-native-web',
+        'react-native$': 'react-native-web',
+        'react-native-web': 'react-native-web',
```

Comment:
This is redundant ig.

Reply:
@shubham1206agra Is there any reason to not update it?

Reply:
Wait
Just remove this line
`'react-native-web': 'react-native-web',`
Sorry my bad. I thought I selected a single line

Reply:
@aimane-chnaif do we want to remove this? i am not sure i want to remove it without knowing the reason why it was added.